### PR TITLE
Fix face orientation

### PR DESCRIPTION
**Pull request by peterrum**
as alternative to #15573

This PR only fixes the face orientations, which is needed for DG. The line orientations are done later (August or September).

@drwells Once merge, could you add P3 for DG?

### CODE REVIEW

**Review by tamiko — APPROVED** [review]

**Review by tamiko — COMMENTED** [review]
Please add a news/changes entry!

**Review by drwells — APPROVED** [review]

**Comment by drwells** [comment]
It feels anticlimactic that this change ended up being so short, but all the simplex tests pass for me (including the new one which verifies that we get the same quadrature points on abutting faces) so I approve. We should definitely add a changelog entry.

File: tests/simplex/orientation_03.cc [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+// Test a mesh with two tetrahedra for all possible orientations. Similar to
+// orientation_02 but also checks that quadrature points on faces (computed via
+// FEFaceValues) are correct.
```
Comment (bangerth) [inline tests/simplex/orientation_03.cc:18]: Nice!

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
Oh, I hit "merge" without waiting for the changelog entry. Please add it -- perhaps in the "major" category.

**Pull request merged**